pyproject.toml

```diff
@@ -38,6 +38,7 @@ dask = { version = "^2021.2.0", optional = true, extras = [
   "array",
   "dataframe"
 ] }
+datafusion = { version = ">=0.4", optional = true }
```

Comment:
Please give this an upper bound, especially since this is pre 1.0. Unbounded upper bounds make it really really easy to start failing CI for hard-to-debug -reasons.

Reply:
bump, this needs an upper bound